src/core/agent/services/keriaNotificationService.ts

```diff
@@ -815,12 +815,23 @@ class KeriaNotificationService extends AgentService {
         const connectionRecord = await this.connectionStorage.findById(
           (operation.response as State).i
         );
-        if (connectionRecord) {
+        const existingConnection = await this.props.signifyClient
```

Comment:
Hey @Sotatek-TungNguyen2a sorry, as I said, my PR is a bit behind getting merged due to causing a breaking change. This change is perfect but can we comment out the logic for `existingConnection` and add a:
```
@TODO: Until https://github.com/WebOfTrust/keripy/pull/882 merged, we can't add this logic (which is meant to stop createdAt from being updated with re-resolves.
Note! Tests need to be added to cover this once uncommented.
```

Reply:
(I haven't noticed a test covering this, so I added it to the TODO comment)

Reply:
@Sotatek-TungNguyen2a No, as I said we need to actually comment it out temporarily.

So have it like this:
```
        // @TODO: Until https://github.com/WebOfTrust/keripy/pull/882 merged, we can't add this logic (which is meant to stop createdAt from being updated with re-resolves.)
        // Also unskip tests for this
        //if (connectionRecord && !existingConnection) {
        if (connectionRecord) {
```

And skip any tests that break with this change.